packages/hooks/use-in-view.ts

```diff
@@ -55,6 +56,10 @@ const useInView: UseInView = ({ rootMargin, onlyOnce } = {}) => {
   const [isIntersecting, setIntersecting] = useState(false);
   const ref = useRef();
 
+  warning(
+    "The @frontity/hooks package is deprecated and will be removed in the future version of frontity.\nThe Frontity team recommends that you use https://github.com/thebuilder/react-intersection-observer instead."
+  );
+
```

Comment:
I'd move this inside a `useEffect` with no deps (`[]`) so we trigger it only once per component.

Reply:
It was not necessary, because we check if the warning was already triggered in the closure. But now I think it's best to just simplify the code, remove the closure and use `useEffect` otherwise, we'd have to keep track of which warnings have been fired (sounds over-engineered to me already)

Reply:
Your right!

Reply:
I put the warning inside of `useEffect` now :)